### PR TITLE
LHC-2788 improve theme uploading process & update homepage for 7.3 release

### PR DESCRIPTION
**Pull request by rdai10**
- Added documentation and update github action to apply upload via github function to prod instance.
- Update homepage for 7.3 release

### CODE REVIEW

File: theme/README.md [hunk 1]
```diff
@@ -36,10 +36,29 @@ Ruby's official site has a clear [instruction page](https://www.ruby-lang.org/en
 -   `npm run format` to format files.
 -   `npm run zip` to zip the theme files, which can be imported into Zendesk.
 
+## Release Theme on Zendesk
+
+There are two ways to release a new theme on Zendesk. Either by uploading a zipped theme or import the theme via github.
```
Comment (jonmak08) [inline theme/README.md:41]: Wordsmith.

`Either by uploading a zipped theme or by importing the theme via github.`

File: theme/README.md [hunk 2]
```diff
@@ -36,10 +36,29 @@ Ruby's official site has a clear [instruction page](https://www.ruby-lang.org/en
...
+To go into the theme workbench interface, sign into _Zendesk Guid_ as an admin, then go into _Custom Design_.
+
+### Upload Zip
+
+-   Click on _Add Theme_ and select _Import Theme_.
+-   Choose the zipped file and upload.
+
+### Import from Github
+
+-   Click on _Add Theme_ and select _Add from Github_
```
Comment (jonmak08) [inline theme/README.md:52]: Missing period at the end of the sentence.

File: theme/README.md [hunk 3]
```diff
@@ -36,10 +36,29 @@ Ruby's official site has a clear [instruction page](https://www.ruby-lang.org/en
...
+-   Enter `rdai10/zendesk` in the first field.
+-   For uploading to sandbox, enter the branch name `sandbox-deploy` in the second field.
```
Comment (jonmak08) [inline theme/README.md:54]: Wordsmith.

For uploading to sandbox, enter the branch name, `sandbox-deploy`, in the second field.

File: theme/README.md [hunk 4]
```diff
@@ -36,10 +36,29 @@ Ruby's official site has a clear [instruction page](https://www.ruby-lang.org/en
...
+-   For uploading to production, enter the branch name `release-deploy` in the second field.
```
Comment (jonmak08) [inline theme/README.md:55]: Wordsmith.

For uploading to production, enter the branch name, `release-deploy`, in the second field.

File: theme/README.md [hunk 5]
```diff
@@ -36,10 +36,29 @@ Ruby's official site has a clear [instruction page](https://www.ruby-lang.org/en
...
+-   Click _Import_ to complete process.
+
 ## Browser Support
 
 -   [Browser support list](http://browserl.ist/?q=>0.25%25%2C+ie+11%2C+not+op_mini+all).
 
 ## Project Documentation
 
-Project documentation is in the [documentation folder](/documentation).
+See more information relating to the project, go to [documentation folder](/documentation).
```
Comment (jonmak08) [inline theme/README.md:64]: Wordsmith.

More information regarding the project can be found in the [documentation folder](/documentation).

**Comment by rdai10** [comment]
Thanks @jonmak08, readme updated at 611f78f!

**Comment by jonmak08** [comment]
LGTM.  Thanks @rdai10!

**Pull request merged**